index.js

```diff
@@ -174,9 +175,7 @@ async function pickRandomFileWithCDN({
       )
     console.log('Picked proof set id:', setId)
 
-    const proofSetLive =
-      cachedProofSetsLive.get(setId) ?? (await pdpVerifier.proofSetLive(setId))
-    cachedProofSetsLive.set(setId, proofSetLive)
+    const proofSetLive = await pdpVerifier.proofSetLive(setId)
```

Comment:
another approach is to keep the cache but update it when we later notice the cache is outdated. Not using a cache at all, as proposed by this PR, is simpler though, so I'm in favor - if we can manage the extra running time